src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.List;
+
 @ToString
```

Comment:
Using @Data annotation from Lombok can lead to generating getters, setters, equals, hashCode, and toString methods, which can violate the Encapsulation principle. (Severity: medium)